### PR TITLE
update RedisExtension

### PR DESCRIPTION
**Pull request by borovHokage**

### CODE REVIEW

**Review by ndoriomedov — APPROVED** [review]

File: src/Kernel.RedisSupport/Extensions/RedisExtension.cs [hunk 1]
```diff
@@ -18,7 +18,17 @@ public static string GetRedisCacheHashCode(this IEnumerable<Guid> guids, params
 
         foreach (var arg in additionalArguments)
         {
-          cache += arg.GetHashCode();
+          if (arg is string)
+          {
+            foreach (char value in arg.ToString().Replace(" ", ""))
+            {
+              cache += cache * 17 + value.GetHashCode();
```
Comment (ndoriomedov) [inline src/Kernel.RedisSupport/Extensions/RedisExtension.cs]: maybe multiply a value hash? when you multiply cache it creates a dependence on additional arguments' order, so if in consumer you set key with additional args like (string, other arg) and in request like (other arg, your string) - they won't be same keys

Comment (ndoriomedov) [reply src/Kernel.RedisSupport/Extensions/RedisExtension.cs]: and i think that you can keep spaces in the string, they have their own hash

File: src/Kernel.RedisSupport/Extensions/RedisExtension.cs [hunk 2]
```diff
@@ -10,6 +10,7 @@ public static string GetRedisCacheHashCode(this IEnumerable<Guid> guids, params
       unchecked
       {
         var cache = 0;
+        var coef = 1;
```
Comment (ndoriomedov) [inline src/Kernel.RedisSupport/Extensions/RedisExtension.cs]: int?
rename to position maybe?

**Review by MariyaSin — APPROVED** [review]

File: src/Kernel.RedisSupport/Extensions/RedisExtension.cs [hunk 3]
```diff
@@ -18,7 +19,18 @@ public static string GetRedisCacheHashCode(this IEnumerable<Guid> guids, params
 
         foreach (var arg in additionalArguments)
```
Comment (VLiamin) [inline src/Kernel.RedisSupport/Extensions/RedisExtension.cs]: May be replaced var?

**Review by VLiamin — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**